examples/braket_features/Native_gates_on_IonQ.ipynb

```diff
@@ -0,0 +1,1666 @@
+{
```

Comment:
<p>Nit: use "circ" or "circuit" rather than circ_id</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>